CIP-????/README.md

```diff
@@ -0,0 +1,469 @@
+---
+CIP: ????
+Title: Structured Contracts
+Category: Ledger/Tools
```

Comment:
```suggestion
Category: Ledger
```
Since we don't have "support" for multiple categories (either on derived web sites or in habitual practice) we should assign this according to the current requirement for the "specific `Value` data structure" currently indicated in _Ledger Spec Preparation_.  If it turns out that structure isn't required then this can revert back to `Tools`.  cc @WhatisRT @lehins

Reply:
This CIP doesn't really affect ledger semantics. We need to implement that data structure at some point anyway to get multi-asset support in conformance testing so I'd say that this CIP has a soft dependency (since structured contracts can already be useful without it) on this being implemented, rather than it being a part of it. So I think it makes more sense in the 'Tools' category.

Reply:
very good, I'll put that change through and will keep an eye out for highlighting that soft dependency (perhaps @polinavino by elaborating on the short item "Instantiate Value" currently in the _Implementation Plan_).

Reply:
p.s. resolved with https://github.com/cardano-foundation/CIPs/pull/963/commits/0789dad2cd4a971caa7bac14854018eb75e25cc8 unless there is any dispute.